src/apyfloatarray_wrapper.cc

```diff
@@ -1011,6 +1011,152 @@ void bind_float_array(nb::module_& m)
             )pbdoc"
         )
 
+        .def(
+            "max",
+            &APyFloatArray::max,
+            nb::arg("axis") = nb::none(),
+            R"pbdoc(
+            Returns the maximum value from an array or the maximum along an axis
+
+            Parameters
+            ----------
+            axis : tuple or int, optional
+                The axis to get the maximum along.
+
+            Returns
+            -------
+            :class:`APyFloatArray` or :class:`APyFloat`
+
+            Raises
+            ------
+            :class:`IndexError`
+                If a specified axis is outside of the existing number of dimensions for the array.
+
+            Examples
+            -------
+
+            >>> from apytypes import APyFloatArray
+
+            Array `a`, array to get the maximum along.
+
+            >>> a = APyFloatArray.from_float(
+            ...     [[1, 2, 3], [4, 5, 6]],
+            ...     exp_bits=10,
+            ...     man_bits=10
+            ... )
+
+            >>> a.max()
+            APyFloat(sign=0, exp=513, man=512, exp_bits=10, man_bits=10)
+
+            >>> a.max(0)
+            APyFloatArray([0, 0, 0], [513, 513, 513], [0, 256, 512], shape=(3,), exp_bits=10, man_bits=10, bias=511)
+
+            >>> a.max(1)
+            APyFloatArray([0, 0], [512, 513], [512, 512], shape=(2,), exp_bits=10, man_bits=10, bias=511)
+
+            -------
```

Comment:
Try to avoid these in future PRs. Better to put another section at the end. (This will render as a `<hline>`.)